api/android/build-android-lib.sh

```diff
@@ -67,22 +67,33 @@ tar xJf ./ext-files/tensorflow-lite-$nnstreamer_tf_lite_ver.tar.xz -C ./api/jni
 sed -i "s|abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'|abiFilters $nnstreamer_target_abi|" api/build.gradle
 
 echo "Starting gradle build for Android library."
+nnstreamer_android_api_lib=./api/build/outputs/aar/api-release.aar
+nnstreamer_android_api_sample=./sample/build/outputs/apk/release/sample-release-unsigned.apk
+
+# Build Android library.
 ./gradlew api:assembleRelease
 
-popd
+# Build sample apk if api library exists.
+if [[ -e $nnstreamer_android_api_lib ]]; then
+    cp $nnstreamer_android_api_lib ./sample/libs
+    ./gradlew sample:assembleRelease
+fi
 
 # Check if build procedure is done.
-nnstreamer_android_api_lib=./build_android_lib/api/build/outputs/aar/api-release.aar
-result_directory=android_lib
+if [[ -e $nnstreamer_android_api_sample ]]; then
+    result_directory=android_lib
+    today=$(date '+%Y-%m-%d')
 
-if [[ -e $nnstreamer_android_api_lib ]]; then
     echo "Build procedure is done, copy NNStreamer library to $result_directory directory."
-    mkdir -p $result_directory
-    cp $nnstreamer_android_api_lib ./$result_directory/nnstreamer-api-$(date '+%Y-%m-%d').aar
+    mkdir -p ../$result_directory
+    cp $nnstreamer_android_api_lib ../$result_directory/nnstreamer-api-$today.aar
+    cp $nnstreamer_android_api_sample ../$result_directory/sample-release-unsigned-$today.apk
```

Comment:
uh.. what is this "sample apk"? is it possible to build an Android app in this git? not in nnstreamer-example.git?

Reply:
I will update this PR to remove warning and code clean.